experimental/IntersectionTheory/src/blowup.jl

```diff
@@ -255,7 +249,7 @@ function blow_up_points(X::AbstractVariety, n::Int; symbol::String = "e")
   if n == 1
     symbs = [SED]
   else
-    symbs = _parse_symbol(SED, 1:n)
+    symbs = ["$SED[$i]" for i in 1:n]
```

Comment:
There is still a `_parse_symbol` method for this

Reply:
Yes, though it is now only is used in a single file, `Main.jl`.